docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,284 @@
+this method break up the sentence into words and punctuation with tag attributes.
+
+
+<a name="lemmatize_get_actual_word"></a>
+## Lemmatizing
+this method use to get actual word from a word.
+
+<a name="language_detect_get_language"></a>
+## Language Detect
+this method use to detect the language of sentence.
+
+
+
+
+## Prerequisites
+
+To enable your application to use the Nlp functionality:
+
+1.  To use the [Tizen.Nlp](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Nlp.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+       <privilege>http://tizen.org/privilege/datasharing</privilege>
+    </privileges>
+    ```
+
+2.  To use the methods and properties of the `Tizen.Network.Nlp` namespace, include it in your application:
+
+    ```
+    using Tizen.Network.Nlp;
+    ```
+
+<a name="word_tokenize_result"></a>
+## To get the tokens of sentence 
+
+To recieve the tokens from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the WordTokenizeAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.WordTokenizeAsync(msg);
+        WordTokenizeResult wr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+<a name="pos_Tag_result"></a>
+## To get the tokens and tags of sentence 
+
+To recieve the tokens and tags from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the PosTagAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnPosButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.PosTagAsync(msg);
+        PosTagResult pr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+<a name="ne_chunk_result"></a>
+## To get the tokens and tags of sentence 
+
+To recieve the tokens and tags from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
```

Comment:
Construct NaturalLanguageProcess on OnCreate(), and connect the Nlp service on init of app: